pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -303,6 +303,8 @@ Reverse lookups can be defined on an entity through the `@derivedFrom` field. Th
 
 For one-to-many relationships, the relationship should always be stored on the 'one' side, and the 'many' side should always be derived. Storing the relationship this way, rather than storing an array of entities on the 'many' side, will result in dramatically better performance for both indexing and querying the subgraph. In general, storing arrays of entities should be avoided as much as is practical.
 
+> Note: You cannot use `@derivedFrom` fields in subgraph mappings. It will compile, but throw an `unexpected null wasm` error during runtime. See: [#219](https://github.com/graphprotocol/graph-ts/issues/219)
```

Comment:
we are actually working on changing this https://github.com/graphprotocol/graph-node/pull/4119

Reply:
removed